src/VisualStudio/CSharp/Test/PersistentStorage/AbstractPersistentStorageTests.cs

```diff
@@ -116,23 +116,11 @@ public async Task TestNullFilePaths()
             Assert.Null(await storage.ReadStreamAsync(document, streamName));
         }
 
-        [Fact]
-        public async Task PersistentService_Solution_WriteReadDifferentInstances()
+        [Theory]
```

Comment:
@heejaechang This change of the pattern does have one implicit behavior change: before the same Solution object was being used for each "underlying test", and now it's creating a new one each time. I suspect that's fine but want your confirmation.

Reply:
ya, it should be fine. it care value to be same not reference.